rb/lib/selenium/webdriver/remote/response.rb

```diff
@@ -58,12 +58,30 @@ def assert_ok
 
         def add_cause(ex, error, backtrace)
           cause = Error::WebDriverError.new
+          backtrace = backtrace_from_remote(backtrace) if backtrace.is_a?(Array)
           cause.set_backtrace(backtrace)
           raise ex, cause: cause
         rescue Error.for_error(error)
           ex
         end
 
+        def backtrace_from_remote(server_trace)
+          server_trace.filter_map do |frame|
+            next unless frame.is_a?(Hash)
+
+            file = frame['fileName']
+            line = frame['lineNumber']
+            meth = frame['methodName']
```

Comment:
```suggestion
            method = frame['methodName']
```

I personally avoid abbreviations.

Reply:
I like method too, I just wanted to preserve the original backtrace_from_remote style